content/riak/kv/3.2.0/setup/installing/alpine-linux.md

```diff
@@ -35,24 +35,28 @@ The following steps have been tested to work with Riak KV on:
 
 * Alpine Linux 3.16 using x86_64
```

Comment:
3.16 got deprecated (no r1 packages) so remove them.

Reply:
Wait, what's the difference?

Reply:
When they got rebuilt, no r1 package was made for 3.16. The r0 is probably fine, but then you have to start explaining to people different package versions for different Alpine OS versions, and it just gets messy.